### PR TITLE
Feature/add shapemodes

### PR DESCRIPTION
**Pull request by meganrm**
I update dataset with updated glossary terms. 
https://docs.google.com/document/d/1pRHFKc1iS07dEzY5FeatlJdNJLq9DRuiZlMuW2qf-ig/edit

@toloudis do you have the cell_feature_analysis json that has the shape mode values? Can you push it to this branch?

### CODE REVIEW

**Comment by toloudis** [comment]
I believe it's already there

File: dataset-2-1/feature_defs.json [hunk 1]
```diff
@@ -134,65 +134,65 @@
...
     "key": "shape-mode-6",
     "discrete": false
   },
   {
-    "displayName": "Shape Mode 7",
-    "description": "Shape Mode 7",
-    "tooltip": "Shape Mode 7",
+    "displayName": "Shape Mode 7 - pear-ness",
+    "description": "All principal components (PCs) were z-scored by dividing the PC values by the standard deviation (σ) of that PC. The z-scored principal components are referred to as “Shape Modes” and the combination of the first 8 Shape Modes creates the 8-dimensional generative shape space. Variation in Shape Mode 7 corresponds to how pear- shaped the cell appears, and the orientation of the “pear” from the top view.",
+    "tooltip": "All principal components (PCs) were z-scored by dividing the PC values by the standard deviation (σ) of that PC. The z-scored principal components are referred to as “Shape Modes” and the combination of the first 8 Shape Modes creates the 8-dimensional generative shape space. Variation in Shape Mode 7 corresponds to how pear- shaped the cell appears, and the orientation of the “pear” from the top view.",
```
Comment (toloudis) [inline dataset-2-1/feature_defs.json:187]: extra space in `pear- shaped` ?

File: dataset-2-1/feature_defs.json [hunk 2]
```diff
@@ -1,56 +1,56 @@
...
+    "tooltip": "The volume of a nucleus of a given cell, as measured by the sum of individual volumes of segmented voxels in the segmented DNA image, in units of \u00b5m\u00b3.",
     "unit": "\u00b5m\u00b3",
     "key": "dna-volume",
     "discrete": false
   },
   {
     "displayName": "DNA surface area",
-    "description": "The surface area of a cell, as measured by segmented pixels and associated edge length, in units of \u00b5m\u00b2.",
-    "tooltip": "The surface area of a cell, as measured by segmented pixels and associated edge length, in units of \u00b5m\u00b2.",
+    "description": "The surface area of a nucleus of a given cell as measured by the sum of individual outer surface areas of segmented voxels (facing the background) in the segmented DNA image, in units of \u00b5m\u00b2.",
```
Comment (toloudis) [inline dataset-2-1/feature_defs.json:44]: nit(s): this seems like it should be `area of the nucleus of a given cell` as opposed to `a nucleus`. Also the DNA height feature doesn't have the phrase "of a given cell".  And some of the texts use "a given cell" and some others just use "a cell".

**Review by toloudis — APPROVED** [review]

**Comment by toloudis** [comment]
> I believe it's already there

I added it. The gitignore tricked me into thinking there were no changes.

File: dataset-2-1/feature_defs.json (hunk 2 above)
Comment (meganrm) [reply dataset-2-1/feature_defs.json:44]: yeah, I'll changed the 'a' to a 'the'. I agree with the "of a given cell" I remove it for the tooltip.

**Review by toloudis — APPROVED** [review]

**Review by schoinh — APPROVED** [review]

**Pull request merged**